src/components/KCard/KCard.vue

```diff
@@ -1,244 +1,116 @@
 
-@import '@/styles/tmp-variables';
-
-.kong-card {
-  background-color: var(--kui-color-background, $kui-color-background);
-  border-radius: var(--kui-border-radius-20, $kui-border-radius-20);
-  padding: var(--kui-space-80, $kui-space-80) var(--kui-space-80, $kui-space-80);
-  transition: box-shadow $tmp-animation-timing-2 ease-in-out;
-
-  &.noBorder {
-    border: none;
-  }
-
-  &.border {
-    border: var(--kui-border-width-10, $kui-border-width-10) solid $tmp-color-black-10;
-    box-shadow: none;
-  }
-
-  &.borderTop {
-    border-top: var(--kui-border-width-10, $kui-border-width-10) solid $tmp-color-black-10;
-  }
-
-  &.hover:hover, &.kcard-shadow {
-    box-shadow: 0 4px 8px $tmp-color-black-10;
-  }
+  display: flex;
+  flex-direction: column;
+  gap: var(--kui-space-70, $kui-space-70);
 
-  .k-card-header {
-    align-items: center;
-    display: flex !important;
-    margin-bottom: var(--kui-space-50, $kui-space-50) !important;
+  .card-header {
+    align-items: flex-start;
+    display: inline-flex;
+    gap: var(--kui-space-50, $kui-space-50);
+    justify-content: space-between;
 
-    &.has-status {
-      align-items: flex-start;
-    }
-  }
-
-  .k-card-status-hat {
-    align-items: center;
-    color: var(--kui-color-text-neutral-stronger, $kui-color-text-neutral-stronger);
-    display: flex;
-    font-size: var(--kui-font-size-20, $kui-font-size-20);
-  }
-
-  .k-card-title {
-    margin-bottom: var(--kui-space-50, $kui-space-50) !important;
-
-    h4 {
+    .card-title {
       color: var(--kui-color-text, $kui-color-text);
-      font-size: var(--kui-font-size-60, $kui-font-size-60);
-      font-weight: var(--kui-font-weight-semibold, $kui-font-weight-semibold);
-      margin: var(--kui-space-0, $kui-space-0);
-      padding: var(--kui-space-0, $kui-space-0);
-    }
-  }
-
-  .k-card-actions  {
-    margin-left: var(--kui-space-auto, $kui-space-auto);
-  }
-
-  .k-card-body {
-    color: var(--kui-color-text-neutral-stronger, $kui-color-text-neutral-stronger);
-    font-size: var(--kui-font-size-20, $kui-font-size-20);
-    line-height: var(--kui-line-height-20, $kui-line-height-20);
-    width: 100%;
-  }
-
-  .k-card-content {
-    display: flex !important;
-
-    .k-table td, table td, :deep(.k-table) td {
-      font-size: var(--kui-font-size-40, $kui-font-size-40);
+      font-size: var(--kui-font-size-50, $kui-font-size-50);
+      font-weight: var(--kui-font-weight-bold, $kui-font-weight-bold);
+      letter-spacing: var(--kui-letter-spacing-minus-30, $kui-letter-spacing-minus-30);
       line-height: var(--kui-line-height-40, $kui-line-height-40);
     }
   }
 
-  .k-card-notifications {
-    margin-left: var(--kui-space-50, $kui-space-50) !important;
-    margin-top: var(--kui-space-auto, $kui-space-auto);
+  .card-content {
+    @include bodyText;
   }
-}
-</style>
 
-<style lang="scss">
-.kong-card {
-  .k-card-header {
-    .k-button {
-      min-height: 38px;
-    }
+  .card-footer {
```

Comment:
question: should the footer be flex by default so items appear inline? Then you could also add a default gap

Reply:
Good idea. Will apply the same to `actions` slot